### DISCUSSION TITLE
How to set disabled status line color?

### DISCUSSION THREAD

**Discussion by ByteDrummer** [nvim-lualine/lualine.nvim#807] [Q&A]
I have the status line disabled for the file tree with `disabled_filetypes = {'NvimTree'}`. And this is how it looks:

<img src=https://i.imgur.com/AT63ukx.png width=600>

I want the light grey, which I outlined, to be dark grey so it matches the color of the file tree. Previously I was able to do this with the following at the end of my vimrc:

```
hi StatusLineNC gui=bold guibg=#21252b guifg=#21252b
hi StatusLine gui=bold guibg=#21252b guifg=#21252b
```

This no longer works with a recent update, and I'm wondering how I could achieve this now.

**Comment by ByteDrummer** [comment]
I switched to the global status line, which works around this and looks better anyways. I'll leave this open just in case, though.

**Comment by shadmansaleh** [comment]
It'll work properly once https://github.com/neovim/neovim/issues/19865 gets fixed .
